src/components/home/index.css

```diff
@@ -34,3 +34,152 @@
   text-align: justify;
   color: #ed2a7a;
 }
+
+.weCare {
+  color: black;
+  margin-left: 150px;
+}
+
+.weCare .pink {
+  color: white;
+  border: none;
+  background-color: deeppink;
+  border-radius: 10px;
+  display: block;
+  font-size: 16px;
+  padding: 12px 28px;
+}
+
+.blue {
```

Comment:
same here for example, the name could be more clear, as it is called "blue" (but actually the color is white ;) )

Reply:
you are right, sorry :) i changed it.